package.json

```diff
@@ -19,32 +19,36 @@
   "author": "",
   "license": "MIT",
   "devDependencies": {
-    "broccoli-asset-rev": "^2.2.0",
+    "broccoli-asset-rev": "^2.4.2",
     "ember-ajax": "0.7.1",
-    "ember-cli": "2.4.1",
+    "ember-cli": "2.5.0",
     "ember-cli-app-version": "^1.0.0",
-    "ember-cli-babel": "^5.1.5",
+    "ember-cli-babel": "^5.1.6",
     "ember-cli-dependency-checker": "^1.2.0",
-    "ember-cli-htmlbars": "^1.0.1",
+    "ember-cli-htmlbars": "^1.0.3",
     "ember-cli-htmlbars-inline-precompile": "^0.3.1",
-    "ember-cli-inject-live-reload": "^1.3.1",
-    "ember-cli-materialize": "0.18.8",
-    "ember-cli-qunit": "^1.2.1",
+    "ember-cli-inject-live-reload": "^1.4.0",
+    "ember-cli-jshint": "^1.0.0",
+    "ember-cli-materialize": "0.19.2",
+    "ember-cli-qunit": "^1.4.0",
     "ember-cli-release": "0.2.8",
-    "ember-cli-sass": "5.3.1",
+    "ember-cli-sass": "git://github.com/mike-north/ember-cli-sass.git#8ceb57d41f5774e8ececb5d1f05454449c19000c",
```

Comment:
Honestly I don't know - this was changed automatically after updating ember. Weird.